src/validation.cpp

```diff
@@ -1560,6 +1600,19 @@ PackageMempoolAcceptResult MemPoolAccept::AcceptMultipleTransactions(const std::
         }
     }
 
+    // Run package-based dust spentness checks
+    if (m_pool.m_opts.require_standard) {
+        if (const auto ephemeral_violation{CheckEphemeralSpends(txns, m_pool.m_opts.dust_relay_feerate)}) {
+            const auto parent_wtxid = ephemeral_violation.value();
```

Comment:
In 872c06dacde6c45c2add69c8c15c94571b08119d "policy: Allow dust in transactions, spent in-mempool"

`CheckEphemeralSpends` appears to only return txids, not wtxid, suggest renaming this variable:

```suggestion
            const auto parent_txid = ephemeral_violation.value();
```

Reply:
done